kaggle_sets/data_preparation/train_test_split.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+from pathlib import Path
+import numpy as np
+
+
+"""
+Usage:
+    dtnp = DatasetToNumpy("mosquito-indicator", ',')
+    x, y = dtnp(["date"], "mosquito_Indicator")
+"""
+
+
+class DatasetToNumpy:
+    def __init__(self, csv_file: str, csv_delimeter: str) -> None:
+        self.csv = Path(f"./datasets/{csv_file}.csv")
+        self.csv_delimeter = csv_delimeter
+
+    def __call__(self, drop_list: list, y_column: str, test_size: float = 0.2, random_seed: int = 42) -> tuple:
```

Comment:
Will it work if we specify test_size = 0?

Reply:
Yes